script/Arcueid/Day 6/06_00_ARC06_2.txt

```diff
@@ -150,6 +154,7 @@ I'm sure you know without me saying that the city has become quite dangerous lat
 [sha:d7fc1e4d19afb49312303b55919deb15416cfcca]{
 -- Page 147, Offset 14279.
 -- 街を騒がせていた連続殺人。
+// to be rechecked
```

Comment:
IIRC ctx was that "series of murders" isn't precisely "serial killings". I kinda think this gets the same thing across, but fine with 
> The serial killings that had the city in an uproar